torch/csrc/jit/fuser/kernel_spec.h

```diff
@@ -127,6 +150,7 @@ struct TORCH_API KernelSpec {
   uint64_t nInputs_;
   std::vector<std::vector<int64_t>> inputBroadcastGroups_;
   std::vector<PartitionInfo> inputChunks_;
+  std::vector<OutputMapAndSize> outputMapAndSizes_;
```

Comment:
This needs docs too. How many of elements does this have? As many as there are tensor inputs?